napari/_qt/widgets/qt_theme_sample.py

```diff
@@ -166,7 +166,7 @@ def screenshot(self, path=None):
         try:
             w = SampleWidget(theme)
         except KeyError:
-            print(f'{theme} is not a recognized theme')
+            print(f'{theme} is not a recognized theme')  # noqa: T201
```

Comment:
Maybe we should print this to stderr?

Reply:
Should I do it also in the other places where we keep `print`?

Reply:
Not everywhere. Some of these prints are debug, not error. 
I would rather think to move them to logger.